frontend/src/services/fetch/endpointConfigs.ts

```diff
@@ -29,7 +29,8 @@ export const userLogoutEndpoint = {
   method: "POST" as ApiMethod,
 };
 
-export const userSavedOpportunityEndpoint = (type: "POST" | "DELETE") => {
+// can expand to support GET when the time comes
+export const toDynamicUsersEndpoint = (type: "POST" | "DELETE") => {
```

Comment:
updated the name since this function can be reused to generate a function for any /user route